applications/dashboard/models/class.usermodel.php

```diff
@@ -2788,7 +2788,7 @@ public function insertForBasic($FormPostValues, $CheckCaptcha = true, $Options =
                 $CaptchaPublicKey = val('Garden.Registration.CaptchaPublicKey', $FormPostValues, '');
                 $CaptchaValid = validateCaptcha($CaptchaPublicKey);
                 if ($CaptchaValid !== true) {
-                    $this->Validation->addValidationResult('Garden.Registration.CaptchaPublicKey', 'The reCAPTCHA value was not entered correctly. Please try again.');
+                    $this->Validation->addValidationResult('Garden.Registration.CaptchaPublicKey', 'The Security Check was not completed correctly. Please try again.');
```

Comment:
I agree with rephrasing this a little, but we can still call it "reCAPTCHA" instead of obfuscating it as a "security check". Also we'll wanna get this into the locales repo ASAP.